app/views/govuk_publishing_components/components/_radio.html.erb

```diff
@@ -1,26 +1,31 @@
 <%
   id_prefix ||= "radio-#{SecureRandom.hex(4)}"
   items ||= []
+
+  is_conditional = false
+  items.each do |item|
+    is_conditional = true if item["conditional"].present?
+  end
 %>
+<% if items.any? %>
```

Comment:
As discussed, this can go because if there are no items it's fine to show something.